src/happy_vllm/routers/technical.py

```diff
@@ -91,6 +91,14 @@ async def get_live_metrics() -> JSONResponse:
     metrics["cpu_cache_usage"] = cpu_cache_usage
     return JSONResponse(metrics)
 
+@router.get("/model_memory_usage", response_model=technical_schema.ResponseModelMemoryUsage)
```

Comment:
Why not put this information in /info ? The memory usage is a model's informations